nms/packages/magmalte/app/views/subscriber/SubscriberTable.js

```diff
@@ -206,69 +207,119 @@ function SubscriberActionsMenu(props: {onClose: () => void}) {
   const enqueueSnackbar = useEnqueueSnackbar();
   const ctx = useContext(SubscriberContext);
   const successCountRef = useRef(0);
+  const [
+    subscriberAction,
+    setSubscriberAction,
+  ] = useState<SubscriberActionType>('add');
   const handleClick = event => {
     setAnchorEl(event.currentTarget);
   };
 
   const handleClose = () => {
     setAnchorEl(null);
   };
+
   /**
-   * Post subscriber chunk or update subscriber(s) if param edit is set to true
+   * Delete Subscribers using a csv file or selecting subscribers with a checkbox in subscriber delete dialog.
```

Comment:
Just a nit - but while this function is used when deleting by CSV or deleting by selecting checkboxes, this function does not require a CSV, or checkboxes. This function just needs the supplied `Array<string>` or `Array<SubscriberInfo`. In my opinion, the function description should not mention CSV or checkboxes, just that it deletes the supplied subscribers.